docs/modules/nifi/partials/supported-versions.adoc

```diff
@@ -2,7 +2,7 @@
 // This is a separate file, since it is used by both the direct NiFi-Operator documentation, and the overarching
 // Stackable Platform documentation.
 
-- 2.0.0-M4 (experimental)
+- 2.0.0-M4 (experimental) - Please note that you need to upgrade to at least 1.27.x before upgrading to 2.0.x!
```

Comment:
I think 1.25 is the magic barrier for 2.0.0 prep.
```suggestion
- 2.0.0-M4 (experimental) - Please note that you need to upgrade to at least 1.25.x before upgrading to 2.0.x!
```

Reply:
https://cwiki.apache.org/confluence/display/NIFI/Migration+Guidance
Actually it's a mess:
1.24 for 2.0.0-M1
1.25 for 2.0.0-M2
1.26 for 2.0.0-M3
1.26 for 2.0.0-M4

WDYT of playing safe and stating 1.27 here?

Reply:
Ah then my knowledge came from the M2... yeah then ignore this and lets play it safe :)